docs/access/vscode.md

```diff
@@ -57,13 +57,15 @@ run the `code` executable that you downloaded the `tunnel` argument.
 You will be asked to choose whether to log in to Microsoft or GitHub (we have tested with GitHub):
 
 ```
-> code tunnel
+> code tunnel --name=$CLUSTER_NAME-tunnel
 ...
 ? How would you like to log in to Visual Studio Code? ›
   Microsoft Account
 ❯ GitHub Account
 ```
 
+* `--name` sets a name for the tunnel, which will be later listed on the VSCode UI
```

Comment:
We could use an admonition:
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:tip
```
!!! tip
    Give the tunnel a unique name using the `--name` flag, which will later be listed on the VSCode UI.
```